package.json

```diff
@@ -43,37 +43,37 @@
   },
   "devDependencies": {
     "@cdk8s/projen-common": "0.0.530",
-    "@cdktf/provider-aws": "^17.0.11",
+    "@cdktf/provider-aws": "^19.0.0",
     "@types/fs-extra": "^11.0.4",
-    "@types/jest": "^27",
+    "@types/jest": "^29",
```

Comment:
Jest added support for typescript 5.x starting with version 29.1.0 ([ts-jest 20.1.0 CHANGELOG.md](https://github.com/kulshekhar/ts-jest/blob/main/CHANGELOG.md#2910-2023-03-26) 2023-03-26)